gen/dcos-config.yaml

```diff
@@ -896,6 +896,11 @@ package:
                   "Port": 5051,
                   "Uri": "/containers",
                   "Role": ["agent", "agent_public"]
+              },
+              {
+                  "Port": 9000,
```

Comment:
this is port 9000 on dcos open... however port 9090 on dcos-ee
https://github.com/mesosphere/dcos-enterprise/blob/master/gen_extra/dcos-config.yaml#L828

I'm also not sure why we are not using {{metronome_port}} as we do with marathon.    Because it should be noted that on strict we don't use either of those ports.  https://github.com/mesosphere/dcos-enterprise/blob/master/gen_extra/dcos-config.yaml#L832

Reply:
This is OSS PR for EE you need to take a look here https://github.com/mesosphere/dcos-enterprise/pull/4193

I agree we should use `{{ metronome_port }}` like [I did for `marathon_port`](https://github.com/dcos/dcos/commit/9b42521dcab8e074166772ccfe790527bfec503c) but it's outside of the scope of this ticket so I leave it up for Marathon team to fix. cc. @meichstedt @alenkacz